File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/02. Mountain Run/02. Mountain Run.js
function mountainRun(record, mt, sec) {
    let seconds = mt * sec
    let add_sec = ~~(mt / 50) * 30
    let subtotal = seconds + add_sec
    let time = subtotal - record

    if (time > subtotal || record > subtotal)
        return `Yes! The new record is ${subtotal.toFixed(2)} seconds.`
    return `No! He was ${time.toFixed(2)} seconds slower.`
}

//console.log(mountainRun(10164, 1400, 25))

File: Python Exams/01. Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/02. Mountain Run/02. Mountain Run.py
import math

record = float(input())
mt = float(input()) 
sec = float(input())
seconds = mt * sec
add_sec = math.floor(mt / 50) * 30
subtotal = seconds + add_sec
time = abs(record - subtotal)

if time > subtotal or record > subtotal:
    print(f'Yes! The new record is {subtotal:.2f} seconds.')
else:
    print(f'No! He was {time:.2f} seconds slower.')



   